### DISCUSSION TITLE
Questions on Azure connection with OpenAI.

### DISCUSSION THREAD

**Discussion by MatheusHRV** [microsoft/AI-Chat-App-Hack#57] [Q&A] [answered]
Hi friends,

I'm a newbie in Azure, i have tree questions regarding the connection with my OpenAI account and GPT models:

1. If i have the paid OpenAI subscription, it offers any benefits on Azure AI services? like a functionality plus or easier ways to connect?
2. Is there any major disadvantages of not using "Azure OpenAI account" Azure service? In other words, wich benefits this specific service provide in the context of our AI chat app project?
3. You know any alternative way to connect other than using high-code with OpenAI API?

Thx.

**Comment by christopher-mierbach** [comment, marked as answer]
Hey MatheusHRV!

1. The two services are not interlinked. So there is no additional benefit in Azure if you have an OpenAI subscription.
2. Regarding the advantages of Azure OpenAI vs. OpenAI, I once saw the following slide at a Microsoft presentation. I think it describes it quite well. Beyond that, and for this project in particular, I don't see any difficulties if you "only" use OpenAI.
![image](https://github.com/microsoft/AI-Chat-App-Hack/assets/153075833/22bc9aca-c70b-4eb4-9116-ba2f26a9a06b)
3. No, unfortunately I am not aware of anything.

**Reply by MatheusHRV** [reply]
Clear!! thank you

**Comment by pamelafox** [comment]
For low code approaches, you could try "Azure OpenAI on Your Data", which was shown briefly in the first two session. OpenAI.com may also have something similar, I'm not sure.

https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/use-your-data

**Reply by MatheusHRV** [reply]
It looks great, but i need Azure OpenAI Service to do this, and i cannot request it (can't use my business email for this purpose), so i'll leave it for now.
Thanks for your response Pamela!!
